apps/events-helsinki/src/pages/api/preview.ts

```diff
@@ -0,0 +1,12 @@
+import { getMaxAge, getToken } from '@events-helsinki/components';
+import type { NextApiRequest, NextApiResponse } from 'next';
+
+export default async (req: NextApiRequest, res: NextApiResponse) => {
+  if (!req.query.secret || !req.query.uri) {
+    return res.status(401).json({ message: 'Invalid token' });
+  }
+
+  const token = req.query.secret.toString();
```

Comment:
`token: getToken(token)` is a bit strange. Could the variable be named "secret" or "token" to be "authToken" and "getAuthToken".

Reply:
fixed